<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KisRecoverNamedAutosaveDialog</class>
 <widget class="QDialog" name="KisRecoverNamedAutosaveDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Krita</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblExplanation">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbOpenAutosave">
     <property name="text">
      <string>Open the autosave file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbDiscardAutosave">
     <property name="text">
      <string>Discard autosave, open the main file</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
